model/src/main/java/com/basistech/rosette/apimodel/recordsimilarity/RecordSimilarityFieldInfo.java

```diff
@@ -31,4 +31,8 @@
 public class RecordSimilarityFieldInfo {
```

Comment:
We actually probably wanted this already, but this should have `@JsonInclude(JsonInclude.Include.NON_NULL)` so that null weights/scoreIfNull values don't show up in the response